### PR TITLE
docs: Adds useTranslation hook example of usage

### PR DESCRIPTION
**Pull request by zarabotaet**
Due
https://github.com/isaachinman/next-i18next/issues/814#issuecomment-688382104

### CODE REVIEW

**Review by isaachinman — APPROVED** [review]

**Comment by isaachinman** [comment]
Needs a semantic title. Thanks @zarabotaet!

**Comment by zarabotaet** [comment]
> Needs a semantic title. Thanks @zarabo

**Comment by zarabotaet** [comment]
> Needs a semantic title. Thanks @zarabotaet!

done

File: README.md [hunk 1]
```diff
@@ -89,6 +89,23 @@ const Footer = ({ t }) => (
...
 export default withTranslation('footer')(Footer)
 ```
+You can use `NextI18Next.useTranslation` hook too! [See in react-i18next docs](https://react.i18next.com/latest/usetranslation-hook)
+
+```jsx
+// This is our initialised `NextI18Next` instance
+import { useTranslation } from '../i18n'
+
+export  const Footer = () => {
+ const {t} = useTranslation('footer')
```
Comment (isaachinman) [inline README.md:99]: Can you add spaces around the `t` destructure?

**Review by isaachinman — CHANGES_REQUESTED** [review]

**Review by isaachinman — APPROVED** [review]

**Pull request merged**